components/Cards/Card.tsx

```diff
@@ -28,7 +28,7 @@ export const Card: FC<CardProps> = ({ data, onClick }) => {
 
   return (
     <article className="z-10 h-full w-full rounded-3xl dark:bg-slate-800 dark:text-text-primary dark:shadow-sm bg-light-white">
-      <div className="card-body">
+      <div className="card-body border-light-silver border-2 border-opacity-75">
```

Comment:
@CBID2, I think we should add the border on the card parent wrapper.

```suggestion
    <article className="z-10 h-full w-full rounded-3xl dark:bg-slate-800 dark:text-text-primary dark:shadow-sm bg-light-white border-light-silver border-2 border-opacity-75">
      <div className="card-body">
````

<img width="1217" alt="Screenshot 2024-03-03 at 11 44 10 AM" src="https://github.com/rupali-codes/LinksHub/assets/93012310/0d186b88-bbce-4cd9-b19f-e5739cea57b8">